### DISCUSSION TITLE
nginx and file-preview config

### DISCUSSION THREAD

**Discussion by arthurkirkosa** [livewire/livewire#1129] [4. General]
```
/livewire/preview-file/35L80fQ6VYkuZw==-.png?expires=1592546824&signature=c756059478488ba9459af520e3d456625eb3f41350fe1163871190003ed4e0e5
```

What configuration does nginx need in order to not try to load the file from disk?
```

  include h5bp/basic.conf;

    location / {
        try_files $uri $uri/ /index.php?$query_string;
    }

    location ~ \.php$ {
        try_files $uri /index.php =404;
        fastcgi_split_path_info ^(.+\.php)(/.+)$;
        fastcgi_pass unix:/run/php/php7.3-fpm.sock;
        fastcgi_index index.php;
        fastcgi_param SCRIPT_FILENAME $document_root$fastcgi_script_name;
        include fastcgi_params;
    }

```

**Comment by animeshkuzur** [comment]
Hey, did you check your nginx `error.log` file? I ran into this same issue, then I checked my nginx `error.log` , apparently it was failing with 
`laravel-app/public/livewire/preview-file/jdBWDogFsPzkpDsqGfUoH2jDwiSwnl-metaZGNscHBmYmZpYWVjaWRoai5wbmc=-.png" failed (2: No such file or directory)`
Livewire uses `storage/app/livewire-tmp` directory to store the temporary files, so just create a symbolic link in your `public/` directory

```bash
$ ln -s $(pwd)/storage/app/livewire-tmp/ $(pwd)/public/livewire/preview-file
``` 
hope this fixes your issue, it worked for me.

**Reply by arthurkirkosa** [reply]
I'll test it and let you know

**Reply by thenibirahmed** [reply]
Still did not fixed can you help?

**Reply by mashb1t** [reply]
see https://github.com/livewire/livewire/issues/1713#issuecomment-703910523

**Reply by arthurkirkosa** [reply]
The symlink route worked for me
